app/assets/stylesheets/webhook_deliveries.css

```diff
@@ -0,0 +1,12 @@
+.response-status-success {
```

Comment:
Just to help draw the eye, light shading based on `response_status`


<img width="1165" alt="Screenshot 2024-09-19 at 5 06 29 PM" src="https://github.com/user-attachments/assets/19ff4f40-65ee-4d2f-bd1e-3181fbfd0e06">